README.md

```diff
@@ -104,12 +104,6 @@ Example command:
 - Twitter: [@marwanEg](https://twitter.com/marwaneg)
 - LinkedIn: [marwan-abdelsattar](https://www.linkedin.com/in/marwan-abdelsattar/)
 
-👤 **oguntayomicheal**
-
-- GitHub: [oguntayomicheal](https://github.com/oguntayomicheal)
-- Twitter: [@Oguns_micky](https://twitter.com/Oguns_micky)
-- LinkedIn: [ogunsmicky](https://www.linkedin.com/in/ogunsmicky/)
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![image](https://github.com/MarwanDev/blog-app/assets/93039359/f27fe87f-9a78-4cdc-8a6d-eaec714f6e63)

- [ ] It would be nice if you fix the highlighted link in the image above because right now it is broken 😨 so kindly fix it to make your readme descriptive and professional 👍